pkg/devconsole/proxy/types.go

```diff
@@ -0,0 +1,15 @@
+package proxy
+
+type RequestBody struct {
```

Comment:
Your request "body" has a field `Body String`. That's a good indicator that this type isn't the full body:

```suggestion
type ProxyRequest struct {
```